spec/request_handler/multipart_parser_spec.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'request_handler/multipart_parser'
+describe RequestHandler::MultipartsParser do
```

Comment:
FYI I probably later look into using MockRequests within the `build_mock_request` instead of the `double` but nothing you have to change right now.